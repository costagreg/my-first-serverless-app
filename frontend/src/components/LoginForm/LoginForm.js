import React from 'react'
import PropTypes from 'prop-types'

import './LoginForm.scss'

const LoginForm = (props) => {
  return (
    <div className="loginForm">
      <form className="loginForm__formContainer">
      <p>Signup to track your link!</p>
        <input className="loginForm__email" placeholder="Email" />
        <input className="loginForm__password" placeholder="Password" />
        <div className="loginForm__buttons">
          <button  className="loginForm__loginButton">Login</button>
          <button  className="loginForm__signupButton">Signup</button>
        </div>
      </form>
    </div>
  )
}

LoginForm.propTypes = {}

export default LoginForm
