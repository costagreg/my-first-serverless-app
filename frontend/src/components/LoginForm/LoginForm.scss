.loginForm {
  display: flex;
  justify-content: center;
  padding: 10px;
  margin: 0 auto;

  &__formContainer {
    display: flex;
    flex-direction: column;
  }

  &__email,
  &__password {
    margin: 0 0 10px 0;
    border: 1px solid #cecece;
    padding: 5px;

  }

  &__buttons {
    display: flex;
    flex-direction: row;


  }

  &__loginButton,
  &__signupButton {
    flex: 1;
    padding: 5px;
  }
}