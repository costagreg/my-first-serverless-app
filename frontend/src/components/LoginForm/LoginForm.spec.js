import React from 'react'
import LoginForm from './LoginForm'
import { render } from '@testing-library/react'

describe('LoginForm', () => {
  describe('@renders', () => {
    it('renders <LoginForm /> without any error', () => {
      const { asFragment } = render(<LoginForm />)

      expect(asFragment()).toMatchSnapshot()
    })
  })
})
