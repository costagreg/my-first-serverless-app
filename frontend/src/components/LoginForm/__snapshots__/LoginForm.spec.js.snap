// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginForm @renders renders <LoginForm /> without any error 1`] = `
<DocumentFragment>
  <div
    class="loginForm"
  >
    <h3>
      Login
    </h3>
    <form
      class="loginForm__formContainer"
    >
      <input
        class="loginForm__email"
        placeholder="Email"
      />
      <input
        class="loginForm__password"
        placeholder="Password"
        type="password"
      />
      <div
        class="loginForm__buttons"
      >
        <button
          class="loginForm__submit"
        >
          Login
        </button>
      </div>
    </form>
  </div>
</DocumentFragment>
`;
